contrib/hack/makeKubectl.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+# Copyright 2016 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+
+# For now pull down a pre-built 'kubectl' image that has the latest
+# build. We need this version in order to pick-up some recent fixes.
+# Once we have an official image we can use then we can delete this section
+docker pull duglin/kubectl
+docker tag duglin/kubectl kubectl
+
+exit 0
+
+# This is the real code we'll keep.
+# Build a 'kubectl' image using the binary from an official build
+
+dir=tmp$RANDOM
+mkdir $dir
+cd $dir
+
+curl -LO https://storage.googleapis.com/kubernetes-release/release/$(curl -s https://storage.googleapis.com/kubernetes-release/release/latest.txt)/bin/linux/amd64/kubectl
+chmod +x kubectl
+
+cat > Dockerfile <<EOF
+FROM scratch
+COPY kubectl /kubectl
+ENTRYPOINT [ "/kubectl" ]
+EOF
+
+docker build -t kubectl
+
+cd ..
+rm -rf $dir
+
+exit 0
+
+# This section will build a new personal kubectl image
```

Comment:
I included this here just so that people can see (and reproduce) how I built duglin/kubectl.  I view this as a temporary thing until we can use the previous section.